docs/sql-ref-syntax-qry-select-hints.md

```diff
@@ -51,6 +51,15 @@ specified, multiple nodes are inserted into the logical plan, but the leftmost h
 
   The `REPARTITION_BY_RANGE` hint can be used to repartition to the specified number of partitions using the specified partitioning expressions. It takes column names and an optional partition number as parameters.
 
+* **REBALANCE_PARTITIONS**
+
+  The `REBALANCE_PARTITIONS` hint can be used to rebalance the query result output partitions, so that
```

Comment:
nit: the other hint docs on this page are just one line, which may be more standard in markdown. Let's follow it and remove \n here.